lib/json_info_codec.dart

```diff
@@ -330,16 +330,11 @@ class AllInfoToJsonConverter extends Converter<AllInfo, Map>
     };
   }
 
-  Map _visitDependencyInfo(DependencyInfo info) =>
-      {'id': info.target.serializedId, 'mask': info.mask};
-
   Map _visitAllInfoHolding(AllInfo allInfo) {
     var map = new SplayTreeMap<String, List>(compareNatural);
     void helper(CodeInfo info) {
       if (info.uses.isEmpty) return;
-      map[info.serializedId] = info.uses
-          .map((u) => _visitDependencyInfo(u))
-          .toList()..sort((Map a, Map b) => compareNatural(a['id'], b['id']));
+      map[info.serializedId] = info.uses.toList();
```

Comment:
I actually prefer to keep `_visitDependencyInfo` here and take `toJson` out of `DependencyInfo` since it's more consistent with the rest of the info objects. This can still be cleaned up a little though...

```dart
map[info.serializedId] = info.uses.map(_visitDependencyInfo).toList();
```

also we aren't sorting anymore?